fresnel/geometry.py

```diff
@@ -183,6 +183,157 @@ def radius(self):
     def color(self):
         return util.array(self._geometry.getColorBuffer(), geom=self)
 
+
+class Box(Cylinder):
+    R""" Box geometry.
+
+    Generate a "box" shape.
+
+    Args:
+        scene (:py:class:`fresnel.Scene`): Add the geometry to this scene
+        box (`numpy.ndarray` or `array_like`): (``1x1, 1x3, or 1x6`` : ``float32``) Assumes 1x1 is cubic, 1x3 is orthorhombic, and 1x6 is triclinic.
+        radius (`numpy.ndarray` or `array_like`): (``N`` : ``float32``) - Radius of box edges.
+        color (`numpy.ndarray` or `array_like`): (``Nx2x3`` : ``float32``) - Color of box edges.
```

Comment:
Similar question to above - should probably be a single color instead of an array of colors.